lib/puma/launcher.rb

```diff
@@ -269,8 +272,16 @@ def prune_bundler
       end
 
       deps = puma.runtime_dependencies.map do |d|
-        spec = Bundler.rubygems.loaded_specs(d.name)
-        "#{d.name}:#{spec.version.to_s}"
+        "#{d.name}:#{spec_for_gem(d.name).version}"
+      end
+
+      Array(@options[:extra_runtime_dependencies]).each do |d_name|
```

Comment:
Hm, how would it be possible for this not to be an Array by this point? Is this coercion necessary?

Reply:
yeah it is, the value's default is `nil` and I think it's easier if we just accept `nil`, a single item, or an array